components/Navbar.tsx

```diff
@@ -0,0 +1,28 @@
+import Link from 'next/link'
+import React from 'react'
+import styles from '../styles/Navbar.module.css'
+
+const Navbar = () => {
+	return (
+		<div className={styles.navbar}>
+		<ul className={styles.listContainer}>
+			<div className={styles.inner}> 
+				<li>
+					<Link className={styles.link} href="/">SKATER LORUM IPSUM</Link>
+				</li>
+				<li>
+					<Link className={styles.link} href="/Sk8rs"> Sk8rs</Link>
+				</li>
+				<li>
+					<Link className={styles.link} href="/Videos">Videos</Link>
```

Comment:
we should use lowercase urls so it should be `/videos` and then the folder inside of pages also `videos`